### ISSUE TITLE
Fine tune flower on inception v3

### ISSUE THREAD

**Issue by sonnyhu** [tensorflow/models#415]
I ran `slim/scripts/finetune_inception_v3_on_flowers.sh`, at the end, got 88% accuracy, compare with old finetune example in inception folder which has 93% accuracy, does anyone have ideas how to make it back to where it were? Thanks.

**Comment by andydavis1** [comment]
This is a question better suited for StackOverflow. Please ask it there and tag it with the `tensorflow` tag.
